layers/+distributions/spacemacs-base/funcs.el

```diff
@@ -588,12 +588,14 @@ dotspacemacs-persistent-server to be t"
 (defun spacemacs/kill-emacs ()
   "Lose all changes and exit Spacemacs"
   (interactive)
+  (run-hook-with-args-until-failure 'kill-emacs-query-functions)
```

Comment:
`kill-emacs-query-functions` can ask the user if he/she really wants to quit Emacs, and in this case we just ignore the answer to the question and quit no matter what. That's weird.